maintainers/team-list.nix

```diff
@@ -569,6 +572,7 @@ with lib.maintainers;
       maralorn
       sternenseemann
       wolfgangwalther
+      peti
```

Comment:
Pretty sure peti was removed from the team on purpose. I don't think adding him would be in his interest, because that involves a lot of notifications.

I last asked about this here: https://github.com/NixOS/nixpkgs/pull/354409#discussion_r2188192960

"honorary member" reads to me as: Let's remove him from the GitHub team. He doesn't maintain these other repos anyway - and if he ever needs access, it will be easy enough to give it to him.

Reply:
This was meant as "if keeping him in GitHub only creates technical problems". We want these lists to be in sync, so he should now be removed from the GItHub team. The same applies to expipiplus1, btw.

Reply:
Clarified in https://github.com/NixOS/nixpkgs/pull/450864#issuecomment-3431029084.

I added peti directly to the cabal2nix and jailbreak-cabal repos. I don't have maintainer rights for the GitHub team itself, so either a maintainer or org owner can remove him there.